### PR TITLE
add service to_background

### PR DESCRIPTION
**Pull request by chpego**
maybe fix this issue #64

### CODE REVIEW

File: custom_components/fullykiosk/sensor.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "batteryLevel": "Battery Level",
     "screenOrientation": "Screen Orientation",
     "foregroundApp": "Foreground App",
+    "backgroundApp": "Background App",
```
Comment (cgarwood) [inline custom_components/fullykiosk/sensor.py:16]: This isn't necessary as Fully doesn't expose a list of background apps

**Review by cgarwood — COMMENTED** [review]
Just one small change. I'll try to get the lib bumped shortly so you can update the manifest too.

File: custom_components/fullykiosk/sensor.py (hunk 1 above)
Comment (chpego) [reply custom_components/fullykiosk/sensor.py]: okay, i remove the line

**Comment by chpego** [comment]
Ok, i remove the last line, but i don't know why checks aren't successful :/

**Comment by cgarwood** [comment]
It's failing the `isort` check:
```
@@ -29,8 +29,8 @@ from .const import (
     SERVICE_RESTART_APP,
     SERVICE_SET_CONFIG,
     SERVICE_START_APPLICATION,
-    SERVICE_TO_FOREGROUND,
     SERVICE_TO_BACKGROUND,
+    SERVICE_TO_FOREGROUND,
 )
 ```

basically it just wants ` SERVICE_TO_BACKGROUND,` moved above ` SERVICE_TO_FOREGROUND,` in the import.

**Comment by cgarwood** [comment]
python-fullykiosk 0.0.11 is publishing now with the updated to background function. That will need updated in the manifest and then we're ready to merge 🙂

**Review by cgarwood — APPROVED** [review]
Looks good!

HACS Validation check failure is a HACS issue, not an issue with this PR.

**Pull request merged**